bookkeeper-server/pom.xml

```diff
@@ -123,6 +123,19 @@
       <groupId>io.reactivex.rxjava3</groupId>
       <artifactId>rxjava</artifactId>
     </dependency>
+    <!-- used on test and main method like `LocalBookKeeper` -->
+    <dependency>
+      <!-- needed by ZooKeeper server -->
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
+      <scope>runtime</scope>
```

Comment:
is this dependency imported to downstream client applications ?
did you test it ?

probably we will need to add "optional"

Reply:
@eolivelli I haven't test it. Let me test it